File: Data Structures & Algorithms/duplicate-integer/submission-0.java

class Solution {
    public static boolean hasDuplicate(int[] nums) {
        Set<Integer> s = new HashSet<>();
        for (int num : nums) {
            s.add(num);
        }
        return s.size() != nums.length; // true if duplicates exist
    }

public static void main(String[] args) {
    System.out.println(hasDuplicate(new int[]{1, 2, 3, 3}));
    }
}

File: Data Structures & Algorithms/sort-an-array/submission-0.py
class Solution:
    def sortArray(self, nums: List[int]) -> List[int]:
        for i in range(len(nums)):
            j=i+1
            for j in range(len(nums)):
                if(nums[j]>nums[i]):
                    temp=nums[i]
                    nums[i]=nums[j]
                    nums[j]=temp
        return nums